src/sentry/static/sentry/app/views/organizationSettings.jsx

```diff
@@ -82,8 +82,16 @@ const OrganizationSettingsForm = React.createClass({
         method: 'PUT',
         data: {
           ...formData,
-          safeFields: formData.safeFields.split('\n'),
-          sensitiveFields: formData.sensitiveFields.split('\n'),
+          safeFields: (
+            (formData.safeFields !== '') ?
+            formData.safeFields.split('\n') :
+            []
+          ),
+          sensitiveFields: (
```

Comment:
This seems that it won't catch say, newlines at the end of input, etc. I see a few of these in the database. Or multiple just empty lines resulting in `[u'', u'']` in storage.